content/08-mongodb/01-what-is-mongodb.mdx

```diff
@@ -0,0 +1,109 @@
+
+Each document can have its own structure with as much or as little complexity as needed. MongoDB provides non-SQL methods and commands to manage and query data programmatically or interactively. MongoDB is known for its fast performance, scalability, and for enabling a rapid development pace.
+
+<PrismaOutlinks>
+
+Prisma's [MongoDB connector](https://www.prisma.io/docs/concepts/database-connectors/mongodb) has recently been [promoted to general availability](https://www.prisma.io/blog/mongodb-general-availability-pixnun6mffmu)!  With this change, you can use the [Prisma Client](https://www.prisma.io/docs/concepts/components/prisma-client) to manage production MongoDB databases with confidence.
+
+Join us in celebrating this milestone on April 25-29 with our [virtual MongoDB Launch week](https://www.prisma.io/mongodblaunch).  There will be exclusive workshops, opportunities for Atlas credits, great swag, and much more!
+
+</PrismaOutlinks>
+
+## Origin story
+
+MongoDB was founded in 2007 by Dwight Merriman, Eliot Horowitz, and Kevin Ryan. They previously worked at an internet advertising company that was serving  400,000 ads per second. 
+
+The team was developing many custom data stores to accommodate this traffic. They regularly ran into struggles around scalability and agility, which led to the inspiration to create a database that solved for these issues. And voila, MongoDB. 
+
+## How it works
+To achieve better scalability and agility, MongoDB works with the JSON variant, Binary JSON, or [BSON](https://www.prisma.io/dataguide/mongodb/mongodb-datatypes).  BSON accommodates more data types than JSON giving more flexibility to the types of data that can be stored in MongoDB.
+
+<PrismaOutlinks>
+
+If you are looking to get started working with MongoDB and Prisma, checkout our [getting started from scratch guide](https://www.prisma.io/docs/getting-started) or how to [add to an existing project](https://www.prisma.io/docs/getting-started/setup-prisma/add-to-existing-project/mongodb-typescript-mongodb).
+
+</PrismaOutlinks>
+
+### MongoDB data Structure
+Rather than tables made up of rows and columns, MongoDB has collections made up of documents. Documents are comprised of field and value pairs. In relation to relational databases, the pieces that make up a MongoDB database can loosely be thought of as follows:
+
+| Relational |   | Document |
+|----------- |---|--------- |
+|Table       | = |Collection|
+|Row | = | Document |
+|Column | =| Field|
+
+The basic structure looks as follows:
+
+```
+{
+   field1: value1,
+   field2: value2,
+   field3: value3,
+   ...
+   fieldN: valueN
+}
+```
+
+Documents look very similar to a JSON object. Documents can hold any of the available BSON data types as well as other documents, arrays, and arrays of documents. This JSON-style format directly maps to native objects in most modern programming languages, making it a natural choice for developers.
+
+ An example would look like:
```

Comment:
Ahh, I guess this is a bit similar to my suggestion above.  I guess if you wanted to do both, the schema here and the relational vs document example I was suggesting could talk about the same general structure.  Just an idea.